amount_helper.rb

```diff
@@ -0,0 +1,54 @@
+module AmountHelper
+  def amount(number, options = {})
+    if number.nil? && options[:nil_as_dash] != false
```

Comment:
this will convert nil always to a dash, even if :nil_as_dash is not true, is this desired?